pkg/suites/driver.go

```diff
@@ -137,119 +137,3 @@ func (d *StateDriver) AssertMultisigState(multisigAddr address.Address, expected
 		assert.Contains(d.tb, multisig.Signers, e, fmt.Sprintf("expected Signer: %v, actual Signer: %v", e, multisig.Signers))
 	}
 }
-
-/*
```

Comment:
removing these commented out methods as too much needs to change for them to be worth keeping around.